openlibrary/components/BulkSearch/components/BulkSearchControls.vue

```diff
@@ -87,8 +87,8 @@ export default {
             this.loadingExtractedBooks = false
         },
         async matchBooks() {
+            this.bulkSearchState.listUrl.resetSeeds()
```

Comment:
Hmmm, I think we might be able to achieve this effect with a bit less state management/logic!

Option 1: Add a `computed`

If we add something like:

```js
{
    computed: {
        listUrl() {
            return 'https://openlibrary.org/lists/add?seeds=' + this.bulkSearchState.matchedBooks
                .map(bm => bm.solrDocs?.docs?.[0]?.key.split('/')[2])
                .filter(key => key);
        }
}
```

This is likely the most vue-y way! The other option is we add a `@click` handler and construct the url on the click, but that can also be a bit messy.

Reply:
Great call